package.json

```diff
@@ -3,6 +3,7 @@
   "private": "true",
   "dependencies": {
     "@hotwired/stimulus": "^3.2.2",
+    "react-router-dom": "^6.20.1",
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4"
```

Comment:
- [ ] As required by the project requirements, I kindly suggest that you please add the `babel-loader` dependency to your `package.json`. The absence of the babel-loader in the package.json causes an error below:

<img width="633" alt="image" src="https://github.com/MarcoDDM/hello-rails-react/assets/52934128/3df6b6cb-988b-40a1-9566-f622f914d24e">